inductiva/molecules/scenarios/water_box/water_box.py

```diff
@@ -0,0 +1,145 @@
+"""Water box scenario."""
+from functools import singledispatchmethod
+from typing import Optional, Literal
+import json
+import os
+import shutil
+
+from inductiva.types import Path
+from inductiva.molecules.simulators import GROMACS
+from inductiva.simulation import Simulator
+from inductiva.utils.templates import (TEMPLATES_PATH,
+                                       batch_replace_params_in_template,
+                                       replace_params_in_template)
+from inductiva.scenarios import Scenario
+from inductiva.utils.files import remove_files_with_tag
+
+SCENARIO_TEMPLATE_DIR = os.path.join(TEMPLATES_PATH, "water_box")
+GROMACS_TEMPLATE_INPUT_DIR = "gromacs"
+
+
+class WaterBox(Scenario):
+    """Water box scenario."""
+
+    def __init__(
+        self,
+        temperature: float = 300,
+        box_size: float = 3.0,
+    ):
+        """The scenario involves simulating a box filled with water molecules.
+        The simulation consists of three main steps: energy minimization, water 
+        molecule position decorrelation, and molecular dynamics simulation. 
+        Currently, this scenario exclusively supports the GROMACS simulator.
+        By default, for a given box size the water molecules are always 
```

Comment:
Is there a way to change this or is it to complicated? In my view, this is the first source of general randomness.

Reply:
Ok, I see this right below. But maybe it should be phrased differently, to notify that, we establish the initial positions uniformly and thereafter apply a decorrelation step to introduce randomness.

Reply:
maybe I can make it more clear! What I meant to say is that for two runs for a box of the same size, the water molecules will be positioned in the same places everytime, this is the default of GROMACS. So to construct the dataset and have multiple experiments for the same box size starting with different positions I have to introduce some randomness in the initial positions.

Reply:
Should the scenario be random, or should the user be able to pass a `seed` to randomize? By this I mean, that running two times the scenario I obtain different results. 

Here, it seems that this will be very focused on the dataset, instead of the general scenarios context. Wdyt, is it possible to have a seed for this?

Reply:
The trajectories from the same starting point will always end up being different, because the simulation is inherently stochastic, due to temperature effects. And I agree that is very focused on the dataset, but the objective was indeed that: I want to repeat the dataset from the article to then be able to create a blogpost and a notebook with it to demonstrate the potential of using the API to create datasets for ML! I don't think that this step of the decorrelation hinders the scenario, but maybe I can redo the description to not be so focused on that small step.

Reply:
I understand your point! Maybe I am confused about the nature of the randomness, since in CFD that does not exist.